xblock/mixins.py

```diff
@@ -22,10 +23,12 @@
 from xblock.internal import class_lazy, NamedAttributesMetaclass
 
 
-XML_NAMESPACES = {
-    "option": "http://code.edx.org/xblock/option",
-    "block": "http://code.edx.org/xblock/block",
-}
+# OrderedDict is used so that namespace attributes are put in predictable order
+# This allows for simple string equality assertions in tests and have no other effects
+XML_NAMESPACES = OrderedDict([
+    ("option", "http://code.edx.org/xblock/option"),
+    ("block", "http://code.edx.org/xblock/block"),
+])
```

Comment:
I don't see anything added in this PR that requires the `OrderedDict`. What tests require this ordering?